gammapy/spectrum/models.py

```diff
@@ -2,24 +2,90 @@
 """Spectral models for Gammapy.
 """
 from __future__ import absolute_import, division, print_function, unicode_literals
+from ..extern.bunch import Bunch
+from . import CountsSpectrum
+import numpy as np
+
 
 __all__ = [
     'SpectralModel',
     'PowerLaw',
     'ExponentialCutoffPowerLaw',
 ]
 
+# Note: Consider to move stuff from _models_old.py here
 
 class SpectralModel(object):
     """Spectral model base class.
     """
+    def __call__(self, energy):
+        kwargs = self.parameters
+        kwargs.update(energy=energy)
+        return self.evaluate(**kwargs)
+
+    def predicted_counts(self, livetime, aeff, edisp):
```

Comment:
I decided to be consistent with ``OGIP`` keywords where exposure means livetime.

 Also, all meta information is stored on the ``on_vector`` = PHA file, since this is where it is stored on disk. It was different in the past, but then the ``meta`` data was scattered all over the place and got out of sync all the time.